services/QuillLMS/spec/helpers/navigation_helper_spec.rb

```diff
@@ -438,4 +438,38 @@
       it { is_expected.to eq false }
     end
   end
+
+  describe '#show_science_dashboard_tab?' do
+    subject { helper.show_science_dashboard_tab?(current_user) }
+
+    context 'when user has unit activities including social studies activities' do
+      before do
+        allow(helper).to receive(:unit_activities_include_science_activities?).and_return(true)
+      end
```

Comment:
This is also organizational, so take or leave it.  The `before` blocks in this context and the following two are very similar, only really differing in the value we want to return.  You could simplify this by doing something like:
```
before do
  allow(helper).to receive(:unit_activities_include_science_activities?).and_return(helper_result)
end

context 'when user has unit activities including social studies activities' do
  let(:helper_result) { true }
  ...
end

context  'when user does not have unit activities including social studies activities' do
  let(:helper_result) { false }
  ...
end
```